### PR TITLE
Update What's New - 1.5.7

### PR DESCRIPTION
**Pull request by torvista**
and a couple more added

Closes #3483

### CODE REVIEW

File: docs/whatsnew_1.5.7.html [hunk 1]
```diff
@@ -173,16 +172,20 @@ <h2>
...
+                <li>Core - Many files have been updated for compatibility with PHP 7.2, 7.3 and 7.4. <b>These changes, along with updating your PHP version, will make your cart faster.</b></li>
+                <li>Admin - The default notification behavior for an Order Status update (send email/do not send email/hide update) may now set by an admin switch</li>
+                <li>Admin - Order Status emails are now sent in the language of the order</li>
+                <li>Admin - Admins can now login to a customer account (login as that customer), to place an order on their behalf</li>
+                <li>Admin - Order Listing. An order-product-details popover column has been added to allow a quick view of the products ordered without opening the order</li>
                 <li>Admin - Validation for integer min and max values and email addresses</li>
-                <li>Admin - Add sort order to order statuses so the admin order update dropdown can mirror your business workflow</li>
-                <li>Admin - Order status updatable from order listing screen</li>
```
Comment (drbyte) [inline docs/whatsnew_1.5.7.html:182]: Was removing this "quick update of order's status" intentional?

File: docs/whatsnew_1.5.7.html [hunk 2]
```diff
@@ -173,16 +172,20 @@ <h2>
...
-                <li>Admin - On-page help added, with links to new documentation</li>
-                <li>Admin – Products to Categories Link Manager. Addition of global tool to copy linked categories to another product.</li>
+                <li>Admin - Added a sort order to the order statuses so the admin order update dropdown can mirror your business workflow</li>
+                <li>Admin - Improved configuration settings descriptions</li>
+                <li>Admin - Help link to the documentation website added to all pages</li>
+                <li>Admin - Attribute Controller – GUI improvements and bugfixes.</li>
+                <li>Admin - Featured – GUI improvements, bugfixes and filtering.</li>
+                <li>Admin – Option Name Manager - GUI improvements, bugfixes and filtering.</li>
+                <li>Admin – Products to Categories Link Manager- GUI improvements, bugfixes and the addition of a global tool to copy linked categories to another product.</li>
+                <li>Admin - Specials – GUI improvements and filtering.</li>
```
Comment (drbyte) [inline docs/whatsnew_1.5.7.html:188]: This list of pages having "GUI improvements" should probably be turned into 1 line listing those pages. Or maybe not even list the pages specifically, and just be generic about it. 
The global-copy-tool addition warrants its own line as it's a new thing. The rest are really typical things that happen between releases so don't need as much mention.

Comment (scottcwilson) [reply docs/whatsnew_1.5.7.html:188]: Please pull in 
 <li>Admin - Products Viewed report and data restructured for performance.</li>

so we can delete my current branch.

File: docs/whatsnew_1.5.7.html (hunk 1 above)
Comment (torvista) [reply docs/whatsnew_1.5.7.html:182]: no

File: docs/whatsnew_1.5.7.html [hunk 3]
```diff
@@ -9,10 +9,9 @@
      */
 //-->
 <head>
-    <title>What's New in Zen Cart&reg; v1.5.7</title>
     <meta charset="utf-8">
+    <title>What's New in Zen Cart&reg; v1.5.7</title>
```
Comment (scottcwilson) [inline docs/whatsnew_1.5.7.html:13]: Suggest change to &tm; here.  I'm trying to do this in other places.

File: docs/whatsnew_1.5.7.html [hunk 4]
```diff
@@ -9,10 +9,9 @@
...
     <meta http-equiv="x-ua-compatible" content="ie=edge">
-    <meta http-equiv="Content-Type" content="text/html">
```
Comment (scottcwilson) [inline docs/whatsnew_1.5.7.html:15]: Why this change?

File: docs/whatsnew_1.5.7.html [hunk 5]
```diff
@@ -86,10 +85,10 @@ <h2>
                 If you are already using v1.5.6 and <em>*have NOT customized or altered*</em> any of the files listed in the changed_files-v1.5.7.html document, then simply replace those files with the new files as listed.
             </p>
             <p>
-                If you have <em>*have customized or altered*</em> certain files, simply re-do your customizations in the new version of those particular files by making the same changes needed.
+                If you have <em>*customized or altered*</em> certain files, you must merge your customizations into the new versions of those files.
```
Comment (scottcwilson) [inline docs/whatsnew_1.5.7.html:88]: Thanks for catching that.

File: docs/whatsnew_1.5.7.html [hunk 6]
```diff
@@ -150,10 +149,10 @@ <h1>
                 With that explained, it is important that whenever you upgrade your site, you should also inspect ALL the template_default files to determine which changes in those files need to be replicated in your customized files.
             </p>
             <p>
-                Often a tool like <a href="https://www.scootersoftware.com/download.php" target="_blank">Beyond Compare</a> or Araxis Merge or GnuDiff can be helpful for this.
+                Comparison/Merging Tools like <a href="https://www.scootersoftware.com/download.php" target="_blank">Beyond Compare</a> or Araxis Merge or GnuDiff are essential for this process.
```
Comment (scottcwilson) [inline docs/whatsnew_1.5.7.html:152]: This change is good. 
Suggestion for further improvement: link to docs page and only mention tools we mention there.
 
Comparison/Merging Tools like <a href="https://www.scootersoftware.com/download.php" target="_blank">Beyond Compare</a> or WinMerge are essential for this process.  See 
 https://docs.zen-cart.com/user/first_steps/useful_tools/#file-comparison-utility

File: docs/whatsnew_1.5.7.html [hunk 7]
```diff
@@ -150,10 +149,10 @@ <h1>
...
             </p>
             <p>
-                The process is simple: compare the template_default directory files from your <em>*old*</em> version against the template_default files in the <em>*new*</em> version, and use those results as a reference to copy those updated changes into each of the files you have customized in your custom/personalized template in your store.
+                The process is simple: compare the template_default directory files from your <em>*old*</em> version to the template_default files in the <em>*new*</em> version, and replicate any differences in the files in your custom/personalized template in your store. </p>
```
Comment (scottcwilson) [inline docs/whatsnew_1.5.7.html:155]: Note: You may also use responsive_classic as a basis for comparison.

File: docs/whatsnew_1.5.7.html [hunk 8]
```diff
@@ -173,16 +172,17 @@ <h2>
...
-                <li>Admin - Order status updatable from order listing screen</li>
-                <li>Admin - Improve many configuration setting descriptions</li>
-                <li>Admin - On-page help added, with links to new documentation</li>
-                <li>Admin – Products to Categories Link Manager. Addition of global tool to copy linked categories to another product.</li>
+                <li>Core - Many files have been updated for compatibility with PHP 7.2, 7.3 and 7.4. <b>These changes, along with updating your PHP version, will make your cart faster.</b></li>
+                <li>Admin - Added a sort order to the Order Statuses so the admin order update dropdown can mirror your business workflow.</li>
+				<li>Admin - The default notification behavior for an Order Status update (send email/do not send email/hide update) may now set by an admin switch.</li>
+                <li>Admin - Order Status may be updated from the order listing screen.</li>
+				<li>Admin - Order Status emails are now sent in the language of the order.</li>
+				<li>Admin - Admins can now login to a customer account (login as that customer), to place an order on their behalf.</li>
```
Comment (scottcwilson) [inline docs/whatsnew_1.5.7.html:180]: Please fix alignment - it's hard to compare otherwise.  I thought this line had been removed.

File: docs/whatsnew_1.5.7.html (hunk 3 above)
Comment (torvista) [reply docs/whatsnew_1.5.7.html:13]: Don't you mean
`&trade;`
?

File: docs/whatsnew_1.5.7.html (hunk 4 above)
Comment (torvista) [reply docs/whatsnew_1.5.7.html:15]: This replaces it
`<meta charset="utf-8">`
and you can't have both.

File: docs/whatsnew_1.5.7.html (hunk 3 above)
Comment (scottcwilson) [reply docs/whatsnew_1.5.7.html:13]: yes thanks.

File: docs/whatsnew_1.5.7.html (hunk 4 above)
Comment (scottcwilson) [reply docs/whatsnew_1.5.7.html:15]: ok

File: docs/whatsnew_1.5.7.html (hunk 8 above)
Comment (torvista) [reply docs/whatsnew_1.5.7.html:180]: ok

File: docs/whatsnew_1.5.7.html [hunk 9]
```diff
@@ -40,22 +39,22 @@
     </section> <!-- End pageHeader //-->
     
     <section id="zenTeamMessage">
-        <h1>Welcome to Zen Cart&reg;</h1>
```
Comment (scottcwilson) [inline docs/whatsnew_1.5.7.html:43]: I would suggest removing the remaining &trade; symbols; doing it once is enough.  It's way overkill to do it every time.

File: docs/whatsnew_1.5.7.html [hunk 10]
```diff
@@ -173,16 +171,17 @@ <h2>
...
-                <li>Admin - Order status updatable from order listing screen</li>
-                <li>Admin - Improve many configuration setting descriptions</li>
-                <li>Admin - On-page help added, with links to new documentation</li>
-                <li>Admin – Products to Categories Link Manager. Addition of global tool to copy linked categories to another product.</li>
+                <li>Core - Many files have been updated for compatibility with PHP 7.2, 7.3 and 7.4. <b>These changes, along with updating your PHP version, will make your cart faster.</b></li>
+                <li>Admin - Added a sort order to the Order Statuses so the admin order update dropdown can mirror your business workflow.</li>
+                <li>Admin - The default notification behavior for an Order Status update (send email/do not send email/hide update) may now set by an admin switch.</li>
+                <li>Admin - Order Status may be updated from the order listing screen.</li>
+				<li>Admin - Order Status emails are now sent in the language of the order.</li>
```
Comment (scottcwilson) [inline docs/whatsnew_1.5.7.html:178]: missed one alignment

File: docs/whatsnew_1.5.7.html (hunk 9 above)
Comment (torvista) [reply docs/whatsnew_1.5.7.html:43]: ok

**Comment by drbyte** [comment]
Thanks!
